content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,247 @@ We have deprecated the following cert-manager APIs:
+```
+
+With the above example, the source label
+`__meta_kubernetes_service_label_app='armada-api'` becomes the new label
+`app='armada-api'` when metrics related to this Service are scraped.
+
+## Akamai issuer and Open Edgegrid EdgeDNS v2
+
+The Akamai issuer has been
+[updated](https://github.com/jetstack/cert-manager/pull/4007) to use Open
+Edgegrid EdgeDNS v2 API.
+
+# Bug Fixes
+
+- The default certificate renewal duration of 30 days was clashing with the
+  duration of certificates issued by the Vault PKI. All Certificates [are
+  now](https://github.com/jetstack/cert-manager/pull/4092) renewed 2/3 through
+  the duration unless a custom renew period is specified with the setting
+  `spec.renewBefore` on the Certificate.
+- The [RFC2136](https://cert-manager.io/docs/configuration/acme/dns01/rfc2136/)
+  issuer is [now](https://github.com/jetstack/cert-manager/pull/3622) able to
+  handle DNS-01 challenges that map to multiple `TXT` records. This lets you
+  create Let's Encrypt certificates using RFC2136 with multiple DNS names.
+- The comparison function `PublicKeysEqual` is
+  [now](https://github.com/jetstack/cert-manager/pull/3914) correct for public
+  keys.
+- The upgrade from v1.3.0 to 1.4.0 with Helm and `--set installCRDs=true` is
+  [now](https://github.com/jetstack/cert-manager/pull/3882) possible. This issue
+  was due to [a Helm
+  bug](https://github.com/helm/helm/issues/5806#issuecomment-788116838) with the
+  `minimum` field on the CRDs.
+- The ACME issuer [now](https://github.com/jetstack/cert-manager/pull/3866)
+  works correctly with Certificates that have a long name (52 characters or
+  more). These Certificates would not get renewed due to non-unique `Order`
+  names being generated.
+- Orders that are used with a misbehaving ACME server should not get stuck
+  [anymore](https://github.com/jetstack/cert-manager/pull/3805). By misbehaving,
+  we mean an ACME server that would validate the authorizations before having
+  set the status of the order to "ready".
+- The internal signers [now](https://github.com/jetstack/cert-manager/pull/3878)
+  set the condition `Ready=False` with the reason `RequestDenied` when a
+  CertificateRequest is `Denied`. This is to keep the same behavior where a
+  terminal state of a CertificateRequest should have a `Ready` condition.
+
+# Honorable mentions
+
+Tim Ramlot ([@inteon](https://github.com/inteon)) has done a fantastic job at
+adding the Istio VirtualService support for HTTP01 challenges in
+[#3724](https://github.com/jetstack/cert-manager/pull/3724). It took an immense
+effort to have this PR ready and merged for the 1.4 release.
+
+After a lot of thinking, we have decided that trying to support every custom
+resource for every proxy could not be done in-tree due to the Go dependency
+weight that each integration adds. Jake Sanders proposed an [out-of-tree
+approach](https://github.com/jetstack/cert-manager/issues/3924), and we have Tim
+[has been
```

Comment:
Suggestion: 'who has been accepted'

Reply:
Should I still apply this change? The sentence is now (a5a3b27):

Tim [has been
accepted](https://summerofcode.withgoogle.com/projects/#4841881566969856) for
the 2021 Google Summer of Code to work on the `kubectl cert-manager install`
command.

Reply:
I think this is correct - @wallrj could confirm that?

Reply:
I'll click "Resolve conversation" for now, feel free to re-open if you think this "Honorable mentions" should not be there @wallrj